frontend/www/js/omegaup/components/user/PasswordEdit.vue

```diff
@@ -64,14 +79,15 @@
 </template>
 
 <script lang="ts">
-import { Vue, Component } from 'vue-property-decorator';
+import { Vue, Component, Prop } from 'vue-property-decorator';
 import T from '../../lang';
 
-@Component({
-  components: {},
-})
+@Component({})
```

Comment:
```suggestion
```

si esto está vacío, se puede eliminar por completo!

Reply:
ah, no sé por qué, pero si quito el `@Component({})` las pruebas no corren, el primer error que marcan es que la variable `T` para las cadenas de traducción es `undefined`

Reply:
ah rayos, me equivoqué, lo que se elimina es nada más el `({})`:

```suggestion
@Component
```

Reply:
oh muy bien